### DISCUSSION TITLE
ollama models are not showing in the UI

### DISCUSSION THREAD

**Discussion by pip-install-skills** [open-webui/open-webui#6167] [Q&A] [answered]
I have install open webui with docker and ollama setup, I already have like 3 models in my ollama list
![image](https://github.com/user-attachments/assets/f6796b09-bd81-4909-99b0-06b9c4642dd3)

In the website UI, I cannot able to see any models even though I can run the ollama models from the terminal
![image](https://github.com/user-attachments/assets/5583d533-5855-4cf2-abe1-4f83a4c1359c)

this is the command I'm using

docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

**Comment by arsaboo** [comment]
Did you add it under Connections?

![image](https://github.com/user-attachments/assets/94b513e3-f492-4ab4-80e1-dfb6426cb69c)

**Reply by pip-install-skills** [reply]
![image](https://github.com/user-attachments/assets/e3b29a41-3f6c-4361-9957-76da3dd9c78b)

It is giving me this error

{
    "detail": "Open WebUI: Server Connection Error"
}

**Reply by pip-install-skills** [reply]
![image](https://github.com/user-attachments/assets/13fdfc53-06e9-4f32-a873-53cf1162820e)

**Reply by pip-install-skills** [reply]
getting this error

{"detail":"WebUI could not connect to Ollama"}

**Comment by pip-install-skills** [comment, marked as answer]
I wasn't running ollama from the docker, but in connections I was using docker url, that is why I wasn't able to see the LLMs even though I have them available locally
